src/App.js

```diff
@@ -380,7 +381,8 @@ export class App extends React.Component<AppProps, AppState> {
             selectedAction: null,
             isDraggingCommand: false,
             audioEnabled: true,
```

Comment:
We should talk about what this variable has meant and should mean moving forward.  Some of the designs had an "all sounds" toggle as well as a "sound effects" toggle, and this most closely corresponds to the latter.  If we end up using a modal and having those two controls, I would suggest adding `audioEffectsEnabled` as well, and using `audioEnabled`  for "all sounds".

Reply:
I agree that we should talk about this. My understanding is that audioEnabled is for "all sounds"